pages/linux/llvd.md

```diff
@@ -0,0 +1,20 @@
+# llvd
+
+> Linkedin Learning Video Downloader.
+> More information: <https://github.com/knowbee/llvd>.
+
+- Download a Course:
+
+`llvd -c "course-slug" --cookies`
```

Comment:
`course-slug` seems to be the part of the URL of the course, it should be wrapped with `{{}}`.

```suggestion
`llvd -c {{course-slug}} --cookies`
```

By the way, can you provide an example of `course-slug`?